File: src/codes/Users_Manual/modulation_1.cpp
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
const auto m = autd3::modulation::Sine(150 * autd3::Hz)
                   .with_sampling_config(autd3::SamplingConfig::Division(5120));
//~return 0; }

File: src/codes/Users_Manual/modulation_1.rs
# extern crate autd3;
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
let m = autd3::modulation::Sine::new(150 * Hz)
            .with_sampling_config(SamplingConfig::Division(5120));
# Ok(())
# }